src/shared/constants/paths/paths.js

```diff
@@ -0,0 +1,9 @@
+import { DETAILS_PAGE, FAVORITES_PAGE, HOME_PAGE } from './pathsNames';
+
+const Routes = {
```

Comment:
Let's name constants in UPPERCASE to make it clear that it's not react Components